src/components/slices/InfoWithImage.js

```diff
@@ -10,13 +10,13 @@ const InfoWithImage = ({ slice }) => {
       <div className="featured-image">
         <picture>
           <source
-            srcSet={featuredImage ? featuredImage.mobile.url : ''}
-            alt={featuredImage ? featuredImage.mobile.alt : ''}
+            srcSet={featuredImage ? featuredImage.url : ''}
+            alt={featuredImage ? featuredImage.alt : ''}
             media="(max-width: 500px)"
           />
           <source
-            srcSet={featuredImage ? featuredImage.tablet.url : ''}
-            alt={featuredImage ? featuredImage.tablet.alt : ''}
+            srcSet={featuredImage ? featuredImage.url : ''}
+            alt={featuredImage ? featuredImage.alt : ''}
```

Comment:
It seems to me that you need to add the image views back in. Otherwise it's pointless to have breakpoints if the image isn't changing.

Reply:
I'll need to look into this as we had to build the repo from scratch and may have missed a few things.

Reply:
Also this is a different plugin. the one this project used before was the graphql one :)

Reply:
I will have to check the queries and see if that property is available  :)